sumo_mgmt.py

```diff
@@ -414,6 +468,52 @@ def add_source(collector_list):
   print_collector_table(collector_list, ['name', 'status', 'description'])  
 
 
+def change_source(collector_list):
+  '''
+  Adds a source specified by the JSON config file argument to the given list of
```

Comment:
This should say "Updates sources having the same name with the given JSON config file argument for the given list of Collectors."